SAKPaaS/src/assets/i18n/de.json

```diff
@@ -104,5 +104,31 @@
     "occupancy-low": "Niedrige Auslastung",
     "occupancy-medium": "Mittlere Auslastung",
     "occupancy-high": "Hohe Auslastung"
+  },
```

Comment:
`user-card.greeting` fehlt noch in der deutschen Version (ich weiß ist nicht deine Schuld, wäre aber nice, wenn du das noch fixen könntest :D)

Reply:
ebenso `footer.impressum` & `footer.privacy-policy`